pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -115,8 +101,13 @@ func NewWorkloadUpdateController(
 	clusterConfig *virtconfig.ClusterConfig,
 ) *WorkloadUpdateController {
 
+	rl := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(time.Duration(defaultThrottleIntervalSeconds)*time.Second, 300*time.Second),
```

Comment:
just to confirm, if the sync fails you don't want there to be a longer wait or back-off between tries?

Reply:
If i'm reading this correctly, a min of 5 seconds exponentially backing off to a max of 5 minutes seems reasonable.

Reply:
Yes, are are correct, my bad. My brain was stuck on seconds and converted the max to 5 seconds instead of 5 minutes.